analyses/doublet-detection/run_doublet-detection-scpca.sh

```diff
@@ -26,8 +26,9 @@ for SAMPLE_DIR in ${DATA_DIR}/${PROJECT_ID}/SCPCS*; do
     mkdir -p ${SAMPLE_RESULTS_DIR}
 
     for SCE_FILE in ${SAMPLE_DIR}/*_processed.rds; do
+        TSV_FILE=`sed 's/.rds/_scdblfinder.tsv/' <<<"$(basename $SCE_FILE)"`
```

Comment:
I know this is less obvious, but extension removal is a builtin bash substitution thing, and in this case it is a bit safer than what you did.

Also, better not to mix backticks and `$()`: you can nest `$()` and it is the preferred method for process substitution.
```suggestion
        TSV_FILE=$(basename "${SCE_FILE%.rds}_scdblfinder.tsv")
```